game.py

```diff
@@ -15,23 +15,23 @@ def play_game(self):
         self.players.append(Player('y'))
         self.board.disp_board()
         while True:
-            try:
-                self.players[turn].get_choice(self.players[turn].name)
+            try:   # while
+                self.players[turn].get_choice(self.players[turn].name) # get_choice returns a value
```

Comment:
Since get_choice returns the column number, you have to store it in a variable, which you didn't do.